views/install.sh.erb

```diff
@@ -121,9 +119,6 @@ elif test -f "/etc/redhat-release"; then
   if test "$platform" = "xenserver"; then
     # Current XenServer 6.2 is based on CentOS 5, platform is not reset to "el" server should hanlde response
     platform="xenserver"
-  else
-    # FIXME: use "redhat"
-    platform="el"
```

Comment:
You can't really, install.sh in omnitruck sucks because it runs in bash on absolutely random operating systems in the wild.  The inputs are pretty indeterminate.

We could do better unit testing of it, but since its an sh script that seems difficult.  I'd prefer to figure out ways of moving install.sh client side mixlib-install in ruby with shelling out individual commands to the target system and incorporating more of dan's boostrapper code, rather than trying to research bash unit testing...

We need less logic here and more logic pushed into platform.rb or pushed into `knife boostrap`/mixlib-install and friends.